src/coreclr/src/tools/Common/JitInterface/CorInfoImpl.cs

```diff
@@ -2798,6 +2793,39 @@ private void setEHcount(uint cEH)
 
         private void setEHinfo(uint EHnumber, ref CORINFO_EH_CLAUSE clause)
         {
+            // Filters don't have class token in the clause.ClassTokenOrOffset
```

Comment:
Maybe we can kick setEHinfo to the r2r specific partial class too? There is not much shared logic anymore.

Reply:
Good point, done.